LinkSDK/LinkWebViewController/LinkWebViewViewController.swift

```diff
@@ -274,18 +274,38 @@ internal extension LinkWebViewViewController {
 
 extension LinkWebViewViewController: WKNavigationDelegate {
     
-    func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Swift.Void) {
-        if let url = navigationAction.request.url {
-            if allowedUrls.contains(where: { url.absoluteString.starts(with: $0) }) {
-                decisionHandler(.cancel)
-                safariViewController = SFSafariViewController(url: url)
-                guard let safariViewController else { return }
-                present(safariViewController, animated: true)
+    func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
+        guard let url = navigationAction.request.url else {
+            decisionHandler(.allow)
+            return
+        }
+
+        // Check if the URL is in allowedUrls (only for http/https)
+        if ["http", "https"].contains(url.scheme),
```

Comment:
is this `["http", "https"].contains(url.scheme)` a redundant condition ?

Reply:
I don't think so, this filters the following logic only to regular URLs which always have to open externally.